### PR TITLE
PLAT-129636: Apply max-width in Arc family components

### PR DESCRIPTION
**Pull request by paul-beldean-lgp**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [x] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [x] Documentation was verified or is not changed
* [x] UI test was passed or is not needed
* [x] Screenshot test was verified or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
slotCenter content was too big and overflowed the arc component

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Added Marquee and max-width to make it display correctly

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
PLAT-129636

### Comments

Enact-DCO-1.0-Signed-off-by: Paul Beldean paul.beldean@lgepartner.com

### CODE REVIEW

File: styles/variables-base.less [hunk 1]
```diff
@@ -225,6 +227,7 @@
 // ---------------------------------------
 @agate-fanspeedcontrol-icon-font-size:   inherit;
 @agate-fanspeedcontrol-icon-height:      inherit;
+@agate-fanspeedcontrol-icon-line-height: 2rem;
```
Comment (daniel-stoian-lgp) [inline styles/variables-base.less:230]: use pixel values in the code. Agate has a resolution independence system that will adapt this value to the screen size.

File: styles/variables-base.less [hunk 2]
```diff
@@ -706,3 +709,4 @@
 @agate-winddirectioncontrol-icon-height: 180px;
 @agate-winddirectioncontrol-icon-line-height: inherit;
 @agate-winddirectioncontrol-icon-width: inherit;
+@agate-winddirectioncontrol-icon-margin: 0px;
```
Comment (daniel-stoian-lgp) [inline styles/variables-base.less:712]: 0 values don't need px

**Comment by daniel-stoian-lgp** [comment]
This PR needs a changelog entry

**Review by daniel-stoian-lgp — APPROVED** [review]

**Review by MikyungKim — CHANGES_REQUESTED** [review]
I agree to add max-width to `slotCenter`, but for Marquee, I hope we could fix qa-a11y sample.

**Review by MikyungKim — APPROVED** [review]
👏 LGTM!

**Review by seunghoh — APPROVED** [review]
LGTM 👍 
The further consideration(even not related to this PR): This is A11y sample but there's no way to read out `slotCenter`. Please consider how can we support to read `slotCenter` with 5-way key when the ArcPicker, ArcSlider get focused.

**Pull request merged**